HorusMessenger.py

```diff
@@ -108,16 +115,16 @@ def process_udp(udp_packet):
 		pass
 
 def udp_rx_thread():
-	global udp_listener_running
```

Comment:
Removing this breaks the setting of this at the end of the file. You need to clean up that other usage.